gspread/spreadsheet.py

```diff
@@ -45,8 +43,25 @@ def url(self):
     @property
     def creationTime(self):
         """Spreadsheet Creation time."""
+        if "createdTime" not in self._properties:
+            self.update_drive_metadata()
         return self._properties["createdTime"]
 
+    @property
+    def lastUpdateTime(self):
+        """Spreadsheet last updated time.
+        Only updated on initialisation.
+        For actual last updated time, use get_lastUpdateTime()."""
+        warnings.warn(
+            """
+            This is only updated on initialisation and is probably outdated by the time you use it.
+            For an up to date last updated time, use get_lastUpdateTime().
+            """
+        )
```

Comment:
should we update the warning and mention something like: deprecated, please use `get_lastUpdateTime()` ?

Reply:
probably, yes. I'll do this.